### PR TITLE
Add an option to use BROWSER env var in addition to CLX_BROWSER

### PR DESCRIPTION
**Pull request by luka-hash**
Cheers :)

### CODE REVIEW

**Comment by bensadeh** [comment]
Hi luka-hash!

Thanks for opening this PR on adding support for the `BROWSER` env variable. For context, the background for adding `CLX_BROWSER` was this issue here: https://github.com/bensadeh/circumflex/issues/79.

The choice of using `CLX_BROWSER` as opposed to `BROWSER` was two-fold:
1. There is no established standard on what the spec is for the BROWSER env variable
2. More importantly, `circumflex` only supports browsers which are not terminal based.

For those reasons, but mostly number 2, I decided to move away from using `BROWSER` because it meant that all environments that had set a terminal browser in their `BROWSER` env, would not be able to open urls in circumflex.

So to sum up, it would be great if circumflex could support the `BROWSER` terminal based ones like `lynx`, `w3m` etc., but since that doesn't work now, I don't want to use `BROWSER` at the moment because the program would break in these circumstances.
